react/components.md

```diff
@@ -0,0 +1,100 @@
+### Introduction
+
+In this lesson we'll be going over the basics of React components - what they do, and how to write them. Make sure to use the project you set up in the [Setting Up A React Environment lesson](https://github.com/TheOdinProject/top-meta/issues/221), but try not to copy and paste any code while you're coding along. 
+
+### Lesson Overview
+
+This section contains a general overview of topics that you will learn in this lesson.
+
+*   What are components?
+*   How are components created?
+*   Where do components live?
+
+### What Are Components
+
+The beauty of React is that it allows you to break a UI (User Interface) down into independent reusable chunks, which we will refer to as components. The following picture should give you an idea of how to do that when building a very basic app. 
+
+![Component Example](./imgs/00.png)
```

Comment:
When this PR is merged, this link needs to be replaced with a statically link. Check out [Adding Images to the Curriculum Wiki entry](https://github.com/TheOdinProject/curriculum/wiki/Adding-Images-to-the-Curriculum)